internal/error.go

```diff
@@ -119,14 +119,28 @@ Workflow consumers will get an instance of *WorkflowExecutionError. This error w
 */
 
 type (
+	// ApplicationErrorOptions represents a combination of error attributes and additional requests.
+	// All fields are optional, providing flexibility in error customization.
+	// NextRetryInterval is a request from server to override retry interval calculated by the
+	// server according to the RetryPolicy set by the Workflow.
+	// IMPORTANT: NextRetryInterval is meaningful only within the context of errors originating from Activity.
+	// It is impossible to specify immediate retry as it is indistinguishable from the default value.
```

Comment:
If users do want immediate retries is the "work around" to just pass a very short timeout?

Reply:
@Quinn-With-Two-Ns you are correct. It is so trivial solution that I did not think it is required to write it down. I can add it.

Reply:
You are correct it is trivial, but sometimes it is nice to point it out anyway for users so they know it is intended. Not a big deal if you don't want to include it.